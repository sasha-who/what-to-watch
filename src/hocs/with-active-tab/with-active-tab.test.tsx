import * as React from "react";
import * as renderer from "react-test-renderer";
import withActiveTab from "./with-active-tab";

const MockComponent = () => {
  return (
    <div></div>
  );
};

const MockComponentWrapped = withActiveTab(MockComponent);

it(`withActiveTab should render correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
