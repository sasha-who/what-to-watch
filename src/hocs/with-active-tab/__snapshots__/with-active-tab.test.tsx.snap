// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveTab should render correctly 1`] = `<div />`;
